src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule73wherejavadocrequired/InputMissingJavadocTypeCheck.java

```diff
@@ -0,0 +1,9 @@
+package com.google.checkstyle.test.chapter7javadoc.rule73wherejavadocrequired;
+
+public class InputMissingJavadocTypeCheck { //warn
+
+}
```

Comment:
please add inner class
as detected at https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/googlestyle-rule73_2020140754/reports/diff/apache-jsecurity/index.html#A6

please add inner class to correct InputMissingJavadocTypeCheckNoViolations.java also.

Reply:
Done.